src/imgui/DebuggableEditor.cc

```diff
@@ -484,6 +485,11 @@ void DebuggableEditor::drawContents(const Sizes& s, Debuggable& debuggable, unsi
 					scrollAddr(s, debuggable, memSize, r2.addr);
 					dataEditingTakeFocus = true;
 				}
+			} else {
+				auto id = ImGui::GetItemID();
+				if (ImGui::Shortcut(manager.getShortcut(ImGuiManager::GOTO_ADDRESS), id, ImGuiInputFlags_RouteGlobalHigh)) {
+					ImGui::SetKeyboardFocusHere(-1);
+				}
```

Comment:
I tried the following (on top of a clean master branch, so without anything from this PR):

On line 489, right after the `if (ImGui::InputText(...))` statment (so fully after it, not in the then and not in the else branch) I added this single line:
`if (ImGui::Shortcut(ImGuiKey_G | ImGuiMod_Ctrl)) std::cerr << "shortcut\n";`
And that seems to work well for me:
* I do see the debug print when the hex-editor-window has focus.
* I don't see any debug prints when the MSX window has focus or when any other ImGui window has focus.

Note:
* I don't think it's needed to pass an `id`.
*  And it's not needed to pass `ImGuiInputFlags_RouteGlobalHigh` (because that's what makes it go wrong for you, I think).

With this fix/simplification, the change in `keybindings.tcl` shouldn't be needed anymore.

Reply:
> I do see the debug print when the hex-editor-window has focus.

I thought you wanted the Ctrl+G shortcut to work even when the hex editor don't have focus. That is how it worked before ImGui, isn't it?

Reply:
And we need the ImGui shortcuts to work when the debugger tool bar doesn't have focus when the user wants to break, continue, step into, step over etc.

Reply:
> For "break, continue, step-in, step-over, ..." it would indeed be a good idea to have these work in multiple windows. At least in the debug toolbar and in the disassembly view. Any others? We can achieve this, without code duplication, by defining a helper function that does the calls to `im::Shortcut()`, and then call that helper from both the debug-toolbar and the disassembly-view code.

I think this is a maintenance nightmare waiting to happen. Who knows what new debug widgets will exist later that need to call a complicated mix of shortcuts that are different from each other because they have different shortcuts that could block each other? I think we should experiment a bit more with the ImGui flags. I think they have potential.